pkg/webhook/v1beta1/pod/utils.go

```diff
@@ -261,6 +261,16 @@ func mutateMetricsCollectorVolume(pod *v1.Pod, mountPath, sidecarContainerName,
 	return nil
 }
 
+func mutatePodContainersEnv(pod *v1.Pod, trial *trialsv1beta1.Trial) {
+	for i := range pod.Spec.Containers {
+		container := &pod.Spec.Containers[i]
+		container.Env = append(container.Env, v1.EnvVar{
+			Name:  "KATIB_TRIAL_NAME",
```

Comment:
Should we make `KATIB_TRIAL_NAME` constants here: https://github.com/kubeflow/katib/blob/ede6e7410c92df359cb79eb01debfeac3507c59d/pkg/controller.v1beta1/consts/const.go#L101-L111?